mesonbuild/interpreter.py

```diff
@@ -2277,7 +2278,11 @@ def func_configure_file(self, node, args, kwargs):
             if not isinstance(inputfile, (str, mesonlib.File)):
                 raise InterpreterException('Input must be a string or a file')
             ifile_abs = os.path.join(self.environment.source_dir, self.subdir, inputfile)
-        elif 'command' in kwargs:
+        elif 'command' in kwargs and not '@INPUT@' in kwargs['command']:
+          pass
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four

<!-- Comment by [SideCI](https://sideci.com) -->